incubator/hnc/pkg/testutils/testutils.go

```diff
@@ -260,6 +260,11 @@ func CleanupNamespaces(nses ...string) {
 		// TODO: remove this line after v0.6 branches and we are no longer supporting v1alpha1
 		// conversion.
 		MustRunWithTimeout(cleanupTimeout, "kubectl annotate ns", ns, "hnc.x-k8s.io/subnamespaceOf-")
+
+		// If it has `ParentMissing` condition, set it as the root namespace
+		MustRun("kubectl hns set --root", ns)
+		// If this is a parent namespace, set allowCascadingDelete 
+		MustRun("kubectl hns set", ns, "-a")
```

Comment:
Do we still need any of this? Now that you've fixed the behaviour of allowCascadingDeletion, we shoudn't need to set it here because we ensure that there aren't any subnamespaces left. And namespaces don't have to be roots to be deleted.

Reply:
You are right, removed